website/addons/twofactor/views.py

```diff
@@ -20,7 +21,8 @@ def user_settings(user_addon, *args, **kwargs):
     if user_addon.verify_code(code):
         user_addon.is_confirmed = True
         user_addon.save()
-        return {'message': 'Successfully verified two-factor authentication.'}, http.OK
+        push_status_message('Successfully verified two-factor authentication!')
```

Comment:
This alert should be green instead of the default warning-yellow since it indicates success.